packages/microcosm-www-next/src/stylesheets/components/_lazy-load.scss

```diff
@@ -0,0 +1,9 @@
+.lazyload,
+.lazyloading {
+	opacity: 0;
+}
+
+.lazyloaded {
+	opacity: 1;
+	transition: opacity 0.3s;
```

Comment:
are these over indented?

Reply:
doesn't look like it in my code, not sure why it's doing that....